README.md

```diff
@@ -35,6 +35,26 @@ For any other action a warning entry is added to log and Record is skipped.
 | `bulkSize`               | The number of items stored in bulk in the index. The minimum value is `1`, maximum value is `10000`. Note that values greater than `1000` may require additional service configuration.                                                          | `true`                                               | `"1000"` |
 | `retries`                | The maximum number of retries of failed operations. The minimum value is `0` which disabled retry logic. The maximum value is `255`. Note that the higher value, the longer it may take to process retries, as a result, ingest next operations. | `true`                                               | `"1000"` |
 
+
+# Source
+ElasticSearch source connector allows you to move data from multiple Elasticsearch indexes with the specified `host` and `indexes`. It uses elasticsearch search api to pull data from indexes. Upon starting it pulls batches of data from indexes, once all the data is retrieved, it then polls the search api to pull data at regular intervals. 
+
+## Configuration Options
+| name                     | description                                                                                                                                                                                                                                      | required                                             | default  |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|----------|
+| `version`                | The version of the Elasticsearch service. One of: `5`, `6`, `7`, `8`.                                                                                                                                                                            | `true`                                               |          |
```

Comment:
Sure, will update this.

Reply:
Here, we will have to take care username/password, API token or other security measires also for fetching cluster info. 
All these are handled by the client library, while creating a new client. So it will be easier to pass the version here instead of calling a secure API endpoint.

Reply:
Good point!